src/Themes/NexusMods.Themes.NexusFluentDark/Resources/Typography/TextBlockControlThemes.axaml

```diff
@@ -36,32 +36,29 @@
         <Setter Property="FontFamily" Value="{StaticResource FontHeadlinesSemiBold}" />
         <Setter Property="FontWeight" Value="SemiBold" />
         <Setter Property="FontSize" Value="36" />
-        <Setter Property="LetterSpacing" Value="0" />
+        <Setter Property="LetterSpacing" Value="-0.9" />
```

Comment:
Negative letter spacing?

Reply:
Yeah, it just removes space from the default amount the font has. If you have this too large in the negative letters start overlapping and it becomes unredable.